tests/sentry/mediators/token_exchange/test_refresher.py

```diff
@@ -42,13 +42,8 @@ def test_deletes_refreshed_token(self):
         self.refresher.call()
         assert not ApiToken.objects.filter(id=self.token.id).exists()
 
-    @patch("sentry.mediators.token_exchange.Validator.run")
-    def test_validates_generic_token_exchange_requirements(self, validator):
-        self.refresher.call()
-
-        validator.assert_called_once_with(
-            install=self.install, client_id=self.client_id, user=self.user
-        )
+    def test_validates_generic_token_exchange_requirements(self):
+        assert self.refresher._validate()
```

Comment:
is this too sus ? Since we're testing/calling a private function ? Lowkey, this is the easiest way to write this test with current design.